toolkit/scripts/containerized-build.mk

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Contains:
+#	- Container env for building RPMs
+
+######## CONTAINER ENV ########
+
+# General targets
+.PHONY: containerized-rpmbuild containerized-rpmbuild-help
+
+containerized_build_args :=
+ifneq ($(MODE),)
+containerized_build_args += -m ${MODE}
+endif
+
+ifneq ($(REPO_PATH),)
+containerized_build_args += -p ${REPO_PATH}
+endif
+
+ifneq ($(VERSION),)
+containerized_build_args += -v ${VERSION}
+endif
+
+ifneq ($(MOUNTS),)
+containerized_build_args += -mo ${MOUNTS}
+endif
+
+ifeq ($(ENABLE_REPO),y)
+containerized_build_args += -r
+endif
+
+containerized-rpmbuild:
```

Comment:
There is an option to use another Mariner repo as the repo for all dev work. (setting REPO_PATH) If we make all targets here, it will end up making it in the current repo, instead of Mariner repo.